src/Modules/UngradedModules/Methods.jl

```diff
@@ -257,21 +248,6 @@ function getindex(r::Hecke.SRow, R::AbstractAlgebra.Ring, u::AbstractUnitRange)
   return s
 end
 
-function getindex(a::Hecke.SRow, b::AbstractVector{Int})
-  if length(a.pos) == 0
-    return a
-  end
-  m = minimum(b)
-  b = sparse_row(parent(a.values[1]))
```

Comment:
Note how the input vector `b` is overwritten here, we only return the minimum value from that vector. I think this is a mistake and one of the two should be renamed if one wanted to fix this.

But I kinda hope nothing calls this method, I'd rather be rid of it here than trying to "fix" it.

Reply:
Yeah that is my guess, too. But I argue that even when untangling `b` and the other `b` by renaming one, the semantics of the resulting code are wrong. E.g. it is legal to do `row[ [1,1,1] ]`  and this should return a list/vector/SRow/whatever of length 3 with three equal entries. The code below doesn't do that at all.

I don't want to waste time trying to fix this. If someone really needs this kind of functionality, it is easier to implement it from scratch (and ideally in Hecke, not here)